Framework/DataHandling/test/LoadEmptyInstrumentTest.h

```diff
@@ -136,7 +149,92 @@ class LoadEmptyInstrumentTest : public CxxTest::TestSuite {
     check_workspace_detectors(output, 64);
   }
 
+  void testLoadFromNexus() {
+    g_log.notice("\ntestLoadFromNexus...");
+
+    LoadEmptyInstrument loader;
+
+    TS_ASSERT_THROWS_NOTHING(loader.initialize());
+    TS_ASSERT(loader.isInitialized());
+
+    TS_ASSERT_THROWS_NOTHING(loader.setPropertyValue("Filename", "PG3_46577.nxs.h5"));
+    TS_ASSERT_THROWS_NOTHING(inputFile = loader.getPropertyValue("Filename"));
+    wsName = "LEIT_LoadFromNexus";
+    TS_ASSERT_THROWS_NOTHING(loader.setPropertyValue("OutputWorkspace", wsName));
+
+    std::string result;
+    TS_ASSERT_THROWS_NOTHING(result = loader.getPropertyValue("Filename"));
+    TS_ASSERT_EQUALS(result, inputFile);
+
+    TS_ASSERT_THROWS_NOTHING(result = loader.getPropertyValue("OutputWorkspace"));
+    TS_ASSERT(!result.compare(wsName));
```

Comment:
Is this using string compare?  Is there a reason the '==' operator cannot be used?  Or `TS_ASSERT_EQUALS`?

Reply:
Yes, that would be cleaner. But it's no longer an issue.